cachecontrol.go

```diff
@@ -29,15 +29,15 @@ func ParseCacheControl(input string) (CacheControl, error) {
 		var token string
 		switch input[pos] {
 		case '"':
-			token = readString(input, pos+1, "\"")
+			token = readUntil(input, pos+1, "\"")
 			pos += len(token) + 1
 		case ',', '\n', '\r', ' ', '\t':
 			continue
 		case '=':
 			isValue = true
 			continue
 		default:
-			token = readString(input, pos, "\"\n\t\r ,=")
+			token = readUntil(input, pos, "\"\n\t\r ,=")
```

Comment:
What about the performance?

I know that many of those operations are implemented in assembly for better performance on most platforms. So it might be a good idea to use it if there's a difference in performance. That's why I suggested it.